sqlx-core/src/postgres/types/network.rs

```diff
@@ -0,0 +1,119 @@
+use std::convert::TryInto;
+use std::net::{Ipv4Addr, Ipv6Addr};
+
+use ipnetwork::{IpNetwork, Ipv4Network, Ipv6Network};
+
+use crate::decode::Decode;
+use crate::encode::Encode;
+use crate::postgres::protocol::TypeId;
+use crate::postgres::row::PgValue;
+use crate::postgres::types::PgTypeInfo;
+use crate::postgres::Postgres;
+use crate::types::Type;
+use crate::Error;
+
+#[cfg(windows)]
+const AF_INET: u8 = 2;
```

Comment:
Is there a better way to get these AF values? Not a huge fan of platform cfg's in here if we can avoid it. If this is the simplest though, it's what it is.

Reply:
Actually, I copy them from diesel directly: https://docs.diesel.rs/src/diesel/pg/types/network_address.rs.html#13.

And this value seems to vary from system to system: https://github.com/postgres/postgres/blob/master/src/include/utils/inet.h#L30-L40